### PR TITLE
Modify Workbook Resource Type to include RG and remove unused endpoint

### PR DESCRIPTION
**Pull request by ericc1103**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by marstr** [comment]
Load balancing my review load out to the very capable, @jianghaolu.

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json [hunk 1]
```diff
@@ -221,93 +261,6 @@
...
-          }
-        },
-        "x-ms-examples": {
-          "WorkbookGet": {
-            "$ref": "./examples/WorkbookGet.json"
-          }
-        }
-      }
-    },
-    "/subscriptions/{subscriptionId}/providers/microsoft.insights/workbooks": {
```
Comment (jianghaolu) [inline specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json]: It's a breaking change to remove an operation here. No breaking changes should be made to an existing stable Swagger.

Comment (ericc1103) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json]: @jianghaolu Thanks for reviewing this. This is new resource type that we just got approved last week. This is not deployed to PROD yet. This is the endpoint we overlooked and needed to be removed.

Comment (jianghaolu) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json]: @ericc1103 Sounds good. When are you deploying to PROD?

Comment (ericc1103) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json]: I am currently working with ARM team and ETA is tomorrow.

**Review by jianghaolu — APPROVED** [review]

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json (hunk 1 above)
Comment (jianghaolu) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json]: @ericc1103 Please add "merge on [date]" on the PR title once you have an ETA. I've approved for now and on the deployment date we can merge.

**Comment by jianghaolu** [comment]
Breaking change but the endpoint hasn't been deployed yet so we will take it. Still waiting for the endpoints to be deployed to merge.

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json (hunk 1 above)
Comment (ericc1103) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json]: @jianghaolu It is now merged in PROD ARM. You can go ahead and approve to merge.

Comment (jianghaolu) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json]: Merging now.

**Pull request merged**